### DISCUSSION TITLE
[FEATURE REQUEST] Background Launch Mode for Streamer Automation Integration

### DISCUSSION THREAD

**Discussion by AshKapow** [howroyd/simonsays#47] [Ideas]
**Is your feature request related to a problem? Please describe.**
Upon launching the application, it takes immediate foreground focus. For streamers, particularly those integrating the application into automation setups (e.g., with a streaming bot), having the application launch in the foreground can disrupt the flow and presentation of their streams.

**Describe the solution you'd like**
I propose an enhancement allowing the application to launch directly in the background. This would entail:

- Launching minimized without taking immediate focus, ensuring a seamless streaming experience.
- Starting directly in the system tray or taskbar without showing its main window.

**Describe alternatives you've considered**
Alternatively, an option could be added to the application's settings, allowing users to choose the startup behaviour: either open in the foreground (current behaviour) or launch in the background (proposed solution).

**Additional context**
The suggested change would significantly enhance the user experience for streamers and content creators, providing a smoother integration into their broadcasting setups and ensuring their streams are not unintentionally interrupted.

**Comment by howroyd** [comment]
Thanks Ash, awesome ideas.

Definitely want to do this and there are a few ways to achieve it.  I shall have to a do a bit of learning to understand how to start minimised unless someone else contributes their knowledge to speed up the process.

Hoping that once Version 2 is done proven stable then progress can be made on this issue.
